WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -988,11 +1002,11 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
         }
     }
 
-    func keyForCurrentListStatusFilter() -> String {
+    class func keyForCurrentListStatusFilter() -> String {
         fatalError("You should implement this method in the subclass")
     }
 
-    func currentFilterIndex() -> Int {
+    class func currentFilterIndex() -> Int {
```

Comment:
If we want to keep this method as a class method, we really need to:

- Unify the instance and class `availablePostListFilters()` methods.  Having two wouldn't be good.
- Have the setter method (`setCurrentFilterIndex()`) become a class method as well.  I know this can be annoying given the current code but these two really need to match, since they're corresponding.

The alternative would be to keep it as an instance method, which in my mind wouldn't be a bad thing either, since setting a "current" something makes the most sense when an instance owns that thing.